view/frontend/web/css/source/_extend.less

```diff
@@ -274,6 +274,21 @@
     }
 }
 
+#payment-method-braintree-lpm {
+    .primary {
+        width: 100%;
+
+        button.action {
+            margin-bottom: 1rem;
+            width: 100%;
+
+            img {
+                height: 4rem;
```

Comment:
Same here, tying an image size to the root font size doesn't seem logical... but there may be some rationale?